### ISSUE TITLE
tapGesture in SideNavigationController.swift cancels out UITableView didSelectRowAtIndexPath

### ISSUE THREAD

**Issue by mattjennings** [CosmicMind/Material#137]
The tapGesture in SideNavigationController.swift cancels out the touch events in the side views. I've only encountered this with a UITableView that I had in the side view controller. didSelectRowAtIndexPath wasn't being called but once I set cancelsTouchesInView to "false" it all worked properly.

Fixed by adding

`tap?.cancelsTouchesInView = false`

to

```
private func prepareGestures(inout pan: UIPanGestureRecognizer?, panSelector: Selector, inout tap: UITapGestureRecognizer?, tapSelector: Selector) {
		removeGestures(&panGesture, tap: &tapGesture)
		
		if nil == pan {
			pan = UIPanGestureRecognizer(target: self, action: panSelector)
			pan!.delegate = self
			view.addGestureRecognizer(pan!)
		}
		
		if nil == tap {
			tap = UITapGestureRecognizer(target: self, action: tapSelector)
            tap?.cancelsTouchesInView = false
			tap!.delegate = self
			view.addGestureRecognizer(tap!)
		}
	}

```

**Comment by danieldahan** [comment]
Excellent, thank you! We will add this in our next release today. I will leave this up until the fix is made, and then notify you and close this issue.

**Comment by danieldahan** [comment]
This is now fixed in 1.29.2, which will be out shortly. Thank you!
